autoload/gitgutter.vim

```diff
@@ -22,12 +21,20 @@ endfunction
 
 
 " Finds the file's path relative to the repo root.
-function! gitgutter#init_buffer(bufnr)
+function! gitgutter#init_buffer(bufnr, ...) abort
```

Comment:
I've tried to keep the API compatible - but if that is not necessary `force` could be required as arg here.